CourseGrab/Keys.swift

```diff
@@ -8,25 +8,14 @@
 
 import Foundation
 
-/**
- `Keys` is a list of expected hidden key value pairs in `Keys.plist`, used for sensitive
- developer information like API keys and secrets.
- */
-enum Keys: String {
-    case googleClientID = "google-client-id"
-
-    /**
-     `value` is the string representation of the key. Implicitly unwrapped because
-     the app should crash without Keys stored properly, while allowing you to check
-     for nil with `value == nil`.
-     */
-    var value: String! {
-        return Keys.keyDict[rawValue] as? String
-    }
+struct Keys {
+    
+    static let googleClientID = Keys.keyDict["google-client-id"] as? String
 
     private static let keyDict: NSDictionary = {
         guard let path = Bundle.main.path(forResource: "Keys", ofType: "plist"),
```

Comment:
Shouldn't the filename be lowercase? (not just in the code but in general)

Reply:
See what you are saying, but GoogleService-Info is also uppercase so I think it's fine

Reply:
Info.plist is also capitalized so I think it's conventional

Reply:
Oh yeah u right